Upgrade from angular 1.3.x to 1.5.8

jdegraft opened the pull request.

dkayiwa commented:
@jdegraft did you just overwrite contents of an existing js file? Or just deleted existing js file and added new one?

jdegraft commented:
My understanding was to replace the angular files. During the upgrade, I renamed underscore.min.js to underscore.min.js.html so I can use a later version of underscore to see if that would resolve the test failures. Later I found the problem was the phantomjs version had to be upgraded in the omod pom so I removed the new underscore.min.js and replaced the original by renaming back. However it still showed as updated in commit.
Summary only. change was to replace angular files, and upgrade phantomjs version in omod pom.

jdegraft commented:
@dkayiwa, I resubmitted the pull request and had the  same result.

It then occurred to me that because of the way git diffs files, unless you have a problem with line endings, the files will never show as completely replaced.

dkayiwa commented on omod/src/main/webapp/resources/scripts/filters/display.js:
```diff
@@ -6,7 +6,7 @@ angular.module('uicommons.filters').
      * If no translation is found, just return input.
      * (If input is an array, these rules will be applied to each element, and they'll be joined with ", ".)
      */
-    filter('omrs.display', function() {
```
Was this also part of the angular upgrade? Do you mind explaining the reason behind it?

dkayiwa commented:
@jdegraft did you try to identify all the uses of omrs.display in dependent modules?

dkayiwa commented:
@jdegraft how is this going?

jdegraft commented:
@dkayiwa, this pull request is complete for uicommons. Can't finish the ticket because module owners for adminui, orderentryui, reportingui have not indicated whether I should change 'ours.display' to 'omrsDisplay' filter in current snapshot or after they release module. This is so that the modules are properly synced with the current new version of uicommons. Your guidance on how to approach this would be helpful.

dkayiwa commented:
@jdegraft did you individually ping them?

jdegraft commented:
[~dkayiwa] just pinged [~wyclif] don't know the module owner of reportingui

dkayiwa commented:
@jdegraft did you try asking on talk who the owners of those other modules are?

jdegraft commented:
[~dakyiwa] [~wyclif] indicates that since orderentryui is not part of the ref app, there is no need to worry about changing it. This is also true for reportingui. Should I just go ahead and make the change for adminui and consider the UICM-72 ticket closeable?

dkayiwa commented:
@jdegraft go ahead because the adminui is already released with its latest changes.

mogoodrich commented:
Reporting UI should be updated... not sure who the owner is, but let me go ahead and do a release of it.
